templates/settings.tpl

```diff
@@ -37,6 +43,69 @@
 				{fbvElement type="checkbox" id="enabledCitationDownloads[]" value=$id checked=in_array($id, $enabledDownloads) label=$style translate=false}
 			{/foreach}
 		{/fbvFormSection}
+		{fbvFormSection list=true title="{if $isApplicationOmp}plugins.generic.citationStyleLanguage.settings.citationUserGroups.omp{else}plugins.generic.citationStyleLanguage.settings.citationUserGroups{/if}"}
+			<p>{if $isApplicationOmp}{translate key="plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription.omp"}{else}{translate key="plugins.generic.citationStyleLanguage.settings.citationUserGroupsDescription"}{/if}</p>
+			<div class="pkp_helpers_quarter inline">
```

Comment:
We're trying not to use these helper classes for half/quarter width. Just put each field on its own line, with the label above it. I'd recommend the following structure:

```
{fbvFormArea ...}
  {fbvFormSection label="..." ...}
    <select>...</select>
  {/fbvFormSection}
  {fbvFormSection label="..." ...}
    <select>...</select>
  {/fbvFormSection}
  ...
{/fbvFormArea}
```